testing/README.md

```diff
@@ -10,13 +10,19 @@ Checkout `ivadomed/.github/workflows/run_tests.yml` to see how tests are run on
 
 ## Running Locally
 
-1. Download the required dataset(s) using the `ivadomed` command line tools:
+1. Install dependencies
+```
+cd ivadomed  # root of the repo
+pip install -e .[dev]
+```
+
+2. Download the required dataset(s) using the `ivadomed` command line tools:
 ```
 cd ivadomed  # root of the repo
 ivadomed_download_data -d data_testing -o data_testing  # for unit tests
 ivadomed_download_data -d data_functional_testing -o data_functional_testing  # for functional tests
 ```
```

Comment:
As Nick suggested, this step is no longer required to run the test suite.

Reply:
I took the liberty of deleting them :)